docs/upgrading_django.md

```diff
@@ -0,0 +1,43 @@
+Upgrading to a newer version of Django
+======================================
+
+It is important to do periodic upgrades of Django to stay up to date with bug fixes, security updates, and get access to new features from newer versions. This document outlines best practices for upgrading to a newer version of Django.
+
+When to do upgrades
+-------------------
+The best strategy for doing upgrades is to stay up to date with the most recent version(s), and to update often. When making frequent upgrades, each update tends to be relatively quick and pain free; trying to make a huge update tends to be less enjoyable.
+For Django's release plan on new versions, check out https://www.djangoproject.com/weblog/2015/jun/25/roadmap/.
+
+Steps
+-----
+We've outlined some steps that have been helpful for us when upgrading Django. You're also welcome to check out Django's docs on upgrading to a newer version https://docs.djangoproject.com/en/dev/howto/upgrade-version/.
+
+1. Check out the release notes for the newer version. For instance, the release notes for Django2.0 can be found here: https://docs.djangoproject.com/en/dev/releases/2.0/. If there is anything from the notes that clearly needs to be updated (a certain function is deprecated, etc.), make those updates to the code. Don't worry about catching everything (sometimes there's not anything that sticks out from the notes).
+
+2. Determine if our code generates any deprecation warnings. Use the -W command line argument when running runserver:
+
+   `heroku local:run python -W manage.py runserver`
```

Comment:
Using the test suite seems like a better option than runserver. Also `-Wonce` can help make the output more manageable to read since it won't show repeated issues.